styles.css

```diff
@@ -6,6 +6,7 @@ body {
 
 header, footer, aside, .main {
   padding: 1em;
+  margin: 0 auto;
```

Comment:
We want this to be applied to all direct children of these four things. That means we need to create a new selector that looks like this...

```css
header, footer, aside, .main > * {
  margin: 0 auto;
}
```
> `>` = direct child
> `*` = everything
> `> *` = apply to all direct children